deepchem/feat/vocabulary_builders/grover_vocab.py

```diff
@@ -25,6 +25,10 @@ class GroverAtomVocabularyBuilder(VocabularyBuilder):
     in the dataset and makes a vocabulary to index mapping by sorting the vocabulary
     by frequency and then alphabetically.
```

Comment:
Can you add a comment explaining what happens when max_size is set in more detail? It looks like the algorithm stops counting new vocab elements once the limit is hit which is reasonable behavior but would be good to document here

Reply:
added documentation